client/mapview_common.cpp

```diff
@@ -1650,7 +1648,6 @@ void decrease_unit_hp_smooth(struct unit *punit0, int hp0,
     }
 
     unqueue_mapview_updates(true);
-    gui_flush();
```

Comment:
Just curious.. Dont we need some sort of repaint here after the units are updated on mapcanvas ?

Reply:
`unqueue_mapview_updates(true)` already calls `flush_dirty()`